src/MusicalScore/Graphical/MusicSheetDrawer.ts

```diff
@@ -45,7 +45,7 @@ export abstract class MusicSheetDrawer {
     public drawingParameters: DrawingParameters;
     public splitScreenLineColor: number;
     public midiPlaybackAvailable: boolean;
-    public drawableBoundingBoxElement: string = process.env.DRAW_BOUNDING_BOX_ELEMENT;
+    public drawableBoundingBoxElement: string = "None"; // process.env.DRAW_BOUNDING_BOX_ELEMENT;
```

Comment:
This is the only one change that I had to do to avoid global variable access to get the tests running.